src/components/Editor/Tabs.css

```diff
@@ -3,7 +3,6 @@
   width: 100%;
   height: 30px;
   display: flex;
-  justify-content: space-between;
```

Comment:
I think we want this for horizontal layout. perhaps we use a media query to only apply this at less than 700px.  

or we add a class to the layout that we can use to select on. I think this approach is simpler. Also we can add docs for doing these style changes for different layouts.

```diff
diff --git a/src/components/App.js b/src/components/App.js
index 81b13be..7dab8dc 100644
--- a/src/components/App.js
+++ b/src/components/App.js
@@ -84,7 +84,7 @@ const App = React.createClass({
     const overflowX = endPanelCollapsed ? "hidden" : "auto";

     return dom.div(
-      { className: "debugger" },
+      { className: "debugger horizontal" },
       SplitBox({
         style: { width: "100vw" },
         initialSize: "300px",
@@ -113,7 +113,7 @@ const App = React.createClass({
     const { horizontal } = this.state;

     return dom.div(
-      { className: "debugger" },
+      { className: "debugger vertical" },
       SplitBox({
         style: { width: "100vw" },
         initialSize: "300px",
```